crates/oxc_prettier/src/printer/mod.rs

```diff
@@ -176,11 +176,14 @@ impl<'a> Printer<'a> {
 
         // TODO: these should be commands
         let mut queue: VecDeque<&Doc<'a>> = doc.iter().collect();
+        let mut cmds = self.cmds.iter().rev();
```

Comment:
I don't see similar lines in https://github.com/prettier/prettier/blob/e3fe2a1f012e5426e007510743454f436cbba9df/src/document/printer.js#L196, can you point me in the right direction?

Reply:
It should be similar to this one.

https://github.com/prettier/prettier/blob/e3fe2a1f012e5426e007510743454f436cbba9df/src/document/printer.js#L219